### PR TITLE
Added challenge hints to challenge cards

### PR DESCRIPTION
**Pull request by weakphish**
Hidden by default with `details` tag.

### CODE REVIEW

File: _includes/challenge_card.html [hunk 1]
```diff
@@ -45,6 +45,28 @@
             </li>
         {% endif %}
 
+        {% if challenge.file_path %}
```
Comment (Ale0x78) [inline _includes/challenge_card.html:48]: Was this supposed to be `challenge.hints` ?

File: _includes/challenge_card.html [hunk 2]
```diff
@@ -45,6 +45,28 @@
...
+                <div class="form-group row">
+                    <div class="col-sm-2">
+                        <strong>Hints </strong>
+                    </div>
+                    <div class="col-sm-5">
+                        <details>
+                            <summary>Click to expand</summary>
+                            {% if challenge.hints %}
+                                {% for hint in challenge.hints %}
+                                    {{ hint }}
```
Comment (Ale0x78) [inline _includes/challenge_card.html:59]: Still will just show all the hints on the same line, unless you wrap them in a <li> or something

**Review by Ale0x78 — APPROVED** [review]
Looks cute!

File: _includes/challenge_card.html (hunk 1 above)
Comment (weakphish) [reply _includes/challenge_card.html:48]: Yeap, thanks

**Pull request merged**